index.bs

```diff
@@ -1943,53 +1947,88 @@ Constructors</h4>
         Sending a <a>control message</a> to start processing means
         executing the following steps:
     
-        <div class="addition proposed" id="c2400-5">
-        <span class="marker">Proposed Addition</span><br>
-            <a href="https://github.com/WebAudio/web-audio-api/issues/2400">Issue
-                2400</a> Access to a different output device
-        <div class="amendment-buttons"></div>
-        <del cite=#2400>
-            1. Attempt to <a href="#acquiring">acquire system resources</a>.
-                In case of failure, abort the following steps.
-
-            3. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
+        <div class="addition proposed">
+            <div class="addition proposed" id="c2400-5" style="border:0; margin:0; padding:0">
+                <span class="marker">Proposed Addition
+                    <a href="https://github.com/WebAudio/web-audio-api/issues/2400">Issue 2400</a>.
+                </span>
+                Access to a different output device
+                <div class="amendment-buttons"></div>
+            </div>
+            <div class="addition proposed" id="c2567-2" style="border:0; margin:0; padding:0">
+                <span class="marker">Proposed Addition
+                    <a href="https://github.com/WebAudio/web-audio-api/issues/2567">Issue 2567</a>.
+                </span>
+                Device-related error reporting via AudioContext
+                <div class="amendment-buttons"></div>
+            </div>
 
-            1. <a href="https://html.spec.whatwg.org/multipage/media.html#queue-a-media-element-task">
-                queue a media element task</a> to execute the following steps:
+            <del cite="#2400 #2567">
+                1. Attempt to <a href="#acquiring">acquire system resources</a>.
+                    In case of failure, abort the following steps.
 
-                1. Set the {{BaseAudioContext/state}} attribute of the {{AudioContext}} to "{{AudioContextState/running}}".
+                3. Set the {{[[rendering thread state]]}} to <code>running</code> on the {{AudioContext}}.
```

Comment:
Will the number there be weird? like 1..3?

Reply:
Fixed, but the numbering is automatically handled by Bikeshed.